EFCore.CheckConstraints/CheckConstraintsExtensions.cs

```diff
@@ -21,12 +21,31 @@ public static class CheckConstraintsExtensions
             return optionsBuilder;
         }
 
+        public static DbContextOptionsBuilder UseDiscriminatorCheckConstraints(
```

Comment:
What is culture used for?

Reply:
Leftover from EFCore.NamingConventions on which this was based, removed.